src/PowerShellEditorServices.Protocol/PowerShellEditorServices.Protocol.csproj

```diff
@@ -3,25 +3,19 @@
   <PropertyGroup>
     <AssemblyTitle>PowerShell Editor Services Host Protocol Library</AssemblyTitle>
     <Description>Provides message types and client/server APIs for the PowerShell Editor Services JSON protocol.</Description>
-    <TargetFrameworks>netstandard1.6;net451;</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <AssemblyName>Microsoft.PowerShell.EditorServices.Protocol</AssemblyName>
   </PropertyGroup>
+  <!--<PropertyGroup>
+    <WarningsAsErrors>1591,1573,1572</WarningsAsErrors>
+    <DocumentationFile>bin\$(TargetFramework)\$(Configuration)\Microsoft.PowerShell.EditorServices.xml</DocumentationFile>
+  </PropertyGroup>-->
   <ItemGroup>
     <ProjectReference Include="..\PowerShellEditorServices\PowerShellEditorServices.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3">
-    </PackageReference>
-    <PackageReference Include="Microsoft.PowerShell.SDK" Version="6.0.0-alpha13">
-    </PackageReference>
-    <PackageReference Include="System.IO.Pipes.AccessControl" Version="4.5.0" Condition=" '$(TargetFramework)' == 'netstandard1.6' " />
-    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
-  </ItemGroup>
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
-    <DefineConstants>$(DefineConstants);CoreCLR</DefineConstants>
-  </PropertyGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
-    <Reference Include="System" />
-    <Reference Include="Microsoft.CSharp" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```

Comment:
I've gotten bitten by this update... we need to support what ever assembly is referenced by PowerShell Core 6.0.x.

Any higher and the binding redirects in place wont work. Double check that PowerShell Core 6.0.x works with this.